docs/_shared/pip-install-determined.txt

```diff
@@ -0,0 +1,3 @@
+.. note::
+    
+    The command, ``pip install determined``, installs the ``determined`` library which includes the Determined command-line interface (CLI). 
```

Comment:
Making a shared component with the note is a nice idea. It's nice, broadly-applicable language, too.